sdks/go/pkg/beam/core/typex/class.go

```diff
@@ -76,6 +78,13 @@ func IsConcrete(t reflect.Type) bool {
 		return false
 	}
 
+	// TODO(wcn): perhaps we need an extension hook to teach this method about
```

Comment:
Perhaps link this TODO to the coder registry jira?

Reply:
Done. Left similar comment in coder.go as well.